apps/demos/Demos/TreeView/DragAndDropHierarchicalDataStructure/React/styles.css

```diff
@@ -33,3 +33,7 @@
 .drive-panel:last-of-type {
   border-right: none;
 }
+
+.dx-sortable {
```

Comment:
This is a workaround to achieve consistent display, but it doesn't address the root cause of the problem. We need to choose an approach here. For example, we can change the element to which we add the margin. In this case, the headers can have a bottom margin instead of the content having a top margin.